common/cosmwasm-smart-contracts/ecash-contract/src/events.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+// event types
+pub const DEPOSITED_FUNDS_EVENT_TYPE: &str = "deposited-funds";
+
+// a 'wasm-' prefix is added to all cosmwasm events
```

Comment:
not unless we set our own event name as opposed to just adding attributes. perhaps we should do that instead?

Reply:
For later

Reply:
Contract storage, TODO later, with dev team help